src/main/scala/firrtl/passes/wiring/WiringUtils.scala

```diff
@@ -125,15 +183,12 @@ object WiringUtils {
     * @return a map of sink instance names to source instance names
     * @throws WiringException if a sink is equidistant to two sources
     */
-  def sinksToSourcesSeq(sinks: Seq[Named],
-                     source: String,
-                     i: InstanceGraph):
-      Seq[(Seq[WDefInstance], Seq[WDefInstance])] = {
+  private[firrtl] def sinksToSourcesSeq(sinks: Seq[Named], source: String, i: InstanceKeyGraph):
+      Seq[(Seq[InstanceKey], Seq[InstanceKey])] = {
```

Comment:
The deprecation on `sinksToSources` points to this, but now it's private?

Reply:
Yeah, since no one was using it and it isn't a pretty API at all. If someone comes and complain we can always make it public again.

Reply:
I guess the issue is that the deprecation on `sinksToSources` points to it, can you change the message there?

Reply:
A clarified the message.